projects/aca-content/src/lib/components/details/details.component.scss

```diff
@@ -3,6 +3,7 @@ app-details-manager {
     display: flex;
 
     .aca-close-details-button {
+      display: flex;
```

Comment:
the parent container is already flex, can you maybe remove the "margin-right" and "margin-top" and use "gap" (or column/row-gap) instead?

Reply:
I tried to do that, but the sibling element to the close button is not another button/simple HTML element, but rather the [aca-toolbar component](https://github.com/Alfresco/alfresco-content-app/blob/9536f24c18ae392ae7af86c4559a782eee94c0a3/projects/aca-content/src/lib/components/details/details.component.html#L16). So just adding the gap properties did not have any effect, and we might have to start modifying the styles of the toolbar component as well, which will have a very high impact, since its being used extensively throughout the application